packages/runtime/container-runtime/src/runningSummarizer.ts

```diff
@@ -264,6 +310,7 @@ export class RunningSummarizer implements IDisposable {
                 summarySequenceNumber: waitStartResult.value.summaryOp.sequenceNumber,
             });
         }
+        this.readyToStartSummarizing = true;
```

Comment:
I'd rename it to something like initialized.
BTW, why we did not need this kind of check before?

Reply:
It was not needed before because we set up the handlers after awaiting `RunningSummarizer.start`. Now, we listen to the op events before we could potentially want to run a heuristic summary.